packages/@adobe/spectrum-css-temp/components/fieldlabel/index.css

```diff
@@ -75,6 +75,14 @@ governing permissions and limitations under the License.
       width: 100%;
       flex: 1 1 auto;
     }
+
+    .spectrum-FieldLabel--quiet {
+      padding-bottom: var(--spectrum-fieldlabel-quiet-padding-bottom);
+      padding-top: var(--spectrum-fieldlabel-quiet-padding-top);
+      line-height: var(--spectrum-fieldlabel-quiet-text-line-height);
+      /* Negative margin so the label is positioned properly */
+      margin-bottom: -3px;
```

Comment:
Shouldn't this come from DNA?

Reply:
I did a search through `dna-vars` after building it locally and couldn't find anything that mentioned a `quiet` variant of the field label. In XD for the 5.3 textfield, the label for quiet textfield is aligned in such a way that the label text has 0 padding between it and the top of the input element, hence why I added these vars.